### PR TITLE
Fix bad link in Theme - distribution - exporting page

### PR DESCRIPTION
**Pull request by ksaandev**

### CODE REVIEW

File: src/content/1.7/themes/distribution/exporting.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ There is no longer any theme data in the database with PrestaShop 1.7. Hence a t
 it's on the disk.
 
 If you want to theme to appears in the backoffice, it's simply have to contain a `config/theme.yml` file.
```
Comment (NeOMakinG) [inline src/content/1.7/themes/distribution/exporting.md:12]:
```suggestion
If you want the theme to appears in the backoffice, it simply has to contain a `config/theme.yml` file.
```

I've some doubt about my english, but it looks like there is some problems there too, I know it's not related to your PR but maybe we can improve this at the same time regarding the fact that you open some PRs to improve typos :)

**Comment by ksaandev** [comment]
@NeOMakinG I know ! I have seen them after I create the PR

I create other PR to fix other issues https://github.com/PrestaShop/docs/pull/571

**Comment by NeOMakinG** [comment]
@ksaandev ok, reviewing the other one then :)

Thanks

File: src/content/1.7/themes/distribution/exporting.md [hunk 2]
```diff
@@ -10,8 +10,8 @@ There is no longer any theme data in the database with PrestaShop 1.7. Hence a t
...
-This will only display it, if you want to select it as your active theme, it has to be valid. Read :doc:`What
-makes a valid theme <testing>`
+This will only display it, if you want to select it as your active theme, it has to be valid. Read ["What
+makes a valid theme"]({{< ref "1.7/themes/distribution/testing/" >}}).
```
Comment (Progi1984) [inline src/content/1.7/themes/distribution/exporting.md:14]:
```suggestion
makes a valid theme"]({{< ref "1.7/themes/distribution/testing.md" >}}).
```

**Comment by matks** [comment]
Thank you @ksaandev

**Pull request merged**